stock_reserve_sale/models/sale.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2013 Camptocamp SA - Guewen Baconnier
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from odoo.exceptions import UserError, except_orm
```

Comment:
```suggestion
from odoo.exceptions import UserError
```

except_orm is a deprecated alias to UserError